packages/clerk-js/src/ui/localization/defaultEnglishResource.ts

```diff
@@ -47,6 +47,7 @@ export const defaultResource: DeepRequired<LocalizationResource> = {
   footerActionLink__useAnotherMethod: 'Use another method',
   badge__primary: 'Primary',
   badge__thisDevice: 'This device',
+  badge__impersonationDevice: 'Impersonator',
```

Comment:
🔧 Can we find a better naming for this as this sounds a bit mysterious?

Reply:
Do you mean the key or the value?

Reply:
The value. Maybe:impersonateed session

Reply:
Resolved via meeting